README.md

```diff
@@ -28,7 +28,7 @@ All commands are given with normal keyboard characters, so those who can type
 with ten fingers can work very fast.  Additionally, function keys can be
 mapped to commands by the user, and the mouse can be used.
 
-Vim runs under MS-Windows (7, 8, 10, 11), macOS, Haiku, VMS and almost all
+Vim runs under MS-Windows (7, 8, 10, 11), macOS, Haiku, VMS, and almost all
```

Comment:
The comma before "and" in a list is optional. So either is fine.

Reply:
that comma before the and, it actually looks wrong according to the rules I was taught at school.

Reply:
Quoting https://justpublishingadvice.com/comma-before-and-in-a-list/

> Do you have to use a comma before and in a list?
> No. If your sentence makes sense and is clear, there is no rule that says you must put a comma.

Quoting https://www.grammarly.com/blog/comma-before-and/

> [...] you have to use a comma after each quality you list except the one that comes immediately before and. That comma is optional.

IMO, it looks nicer without the comma before "and".

Reply:
And, it's always wrong if there are only two elements in the list.